src/app/schedule/schedule-edit/schedule-edit.component.ts

```diff
@@ -27,19 +26,19 @@ import { Settings } from '../../models/settings.model';
   styleUrls: ['./schedule-edit.component.css']
 })
 export class ScheduleEditComponent implements OnDestroy, OnChanges {
-  public schedule: ScheduleDay[];
+  @Input() settings: Settings | undefined;
+  @Output() isPrintViewChanged = new EventEmitter<boolean>(); // fired when isPrintView turned on/off
+
+  public schedule: ScheduleDay[] = [];
   public isPrintView = false;
-  public scheduleForm: FormGroup;
+  public scheduleForm: FormGroup | undefined;
```

Comment:
Pewnie wiesz ale możesz uprościć zapis z undefined do
`public scheduleForm?: FormGroup;`

Ale to już stosuj wg własnego uznania :)